### DISCUSSION TITLE
Online Ebook Maker

### DISCUSSION THREAD

**Discussion by Anishgupta0** [scaleracademy/open-source-projects#115] [Ideas]
Online E-Book Maker Project will allow people to make E-Book for free. Admin and Author logins will be available for this software. The authors will be able to create their E-Books using this system without any hassle or cost. Author will have to register if new member in the system. And existing member can Create new book or resume with old book. One Author cannot keep more than 3 incomplete Books, he has to complete 1 book atleast to start new one. Book will be created on basics of Title, Context, No of Pages in books. 
###	Admin 
- Accept request from Author
- Check Author Details
- Checked for finished Books
- Make E-Books
- Can Process request by mailing the E-Books to authors

### Author
- Register 
- Fill Details
- Login and Choose 
- New E-book or Resume Existing
- Enter Context of book 
- Can pause e-book after completing context
- And resume it.
- Can have E-Book in Word Or PDF format.
### Advantages:
- Fast and easy way of to create E-Books.
- Admin can view E-Books.
- Admin can send email for any E-Book request.
- Authors can take break after completing Context and can even Edit it back
- Author can select no of Context, Add Book Cover. 

### Application:
This system can be used by author, who want to create their books and spread it to world.

**Comment by iamrahul8** [comment]
Good idea 🙌
